doc/user/content/ingest-data/mongodb/_index.md

```diff
@@ -0,0 +1,336 @@
+
+- **Debezium Connect:** Version 2.5.4+ or later. A Kafka Connect cluster with
+  the MongoDB connector plugin installed.
+
+## A. Create a Debezium user in MongoDB.
+
+{{< important >}}
+For Debezium to capture changes, your MongoDB deployment must be configured as a
+replica set or a sharded cluster.
+{{< /important >}}
+
+Create a user (e.g., `debezium_materialize_user`) with the necessary permissions
+to read from the database. Depending on the connector's
+[`capture.scope`](https://debezium.io/documentation/reference/stable/connectors/mongodb.html#required-user-permissions)
+property, the user may need `read` on a specific database or all databases.
+
+This example uses `capture.scope` of `database` to read from the `test` database
+only:
+
+```javascript
+db.getSiblingDB("test").createUser({
+    user: "debezium_materialize_user",
+    pwd: passwordPrompt(),         // Or  "<cleartext password>"
+    roles: [
+      { role: "read", db: "test" }
+    ]
+});
+```
+
+For simplicity, the user is created in the `test` database; however, the user
+can be created in a different database.
+
+## B. Configure the streaming layer
+
+The streaming layer requires Kafka, Schema Registry, and Kafka Connect with the
+Debezium MongoDB connector. If you have not set up these services, see:
+
+- [Confluent Platform
+  Quickstart](https://docs.confluent.io/platform/current/platform-quickstart.html)(*Recommended
+  for Testing*): Docker-based setup  for Kafka, Schema Registry, and Kafka
+  Connect.
+
+- [Debezium
+    Tutorial](https://debezium.io/documentation/reference/stable/tutorial.html):
+    Example using Docker Compose.
+
+### 1. Configure Kafka Connect worker
+
+Configure your Kafka Connect worker to use Avro serialization for Schema
+Registry integration; specifically, set the following environment
+variables/properties are set on your Connect worker:
+
+| Property | Value |
+|----------|-------|
+| `KEY_CONVERTER` | `io.confluent.connect.avro.AvroConverter` |
+| `VALUE_CONVERTER` | `io.confluent.connect.avro.AvroConverter` |
+| `CONNECT_KEY_CONVERTER_SCHEMA_REGISTRY_URL` | `http://schema-registry:8081` |
+| `CONNECT_VALUE_CONVERTER_SCHEMA_REGISTRY_URL` | `http://schema-registry:8081` |
+
+### 2. Configure the Debezium connector for MongoDB
+
+Register the MongoDB connector with the following configuration:
+
+| Parameter | Description |
+|-----------|-------------|
+| `name` | A unique name for the connector instance. |
+| [`connector.class`](https://docs.confluent.io/kafka-connectors/debezium-mongodb-source/current/mongodb_source_connector_config.html) | The Debezium MongoDB connector class, `"io.debezium.connector.mongodb.MongoDbConnector"`. |
+| [`mongodb.connection.string`](https://debezium.io/documentation/reference/stable/connectors/mongodb.html#mongodb-property-mongodb-connection-string) | Your [MongoDB connection string](https://www.mongodb.com/docs/manual/reference/connection-string-formats/) with the `debezium_materialize_user`.  |
```

Comment:
Just fyi -- more for our edification:

- Although kafka connector states "To use this option, you must set the value of mongodb.members.auto.discover to true" https://docs.confluent.io/kafka-connectors/debezium-mongodb-source/current/mongodb_source_connector_config.html
- Debezium docs in v2.3 states:  "Due to changes in the way that the connector manages database connections, this release of Debezium no longer supports use of the mongodb.members.auto.discover property to prevent the connector from performing membership discovery." https://debezium.io/documentation/reference/2.3/connectors/mongodb.html#supported-mongodb-topologies